py/torch_tensorrt/dynamo/conversion/impl/elementwise/ops.py

```diff
@@ -422,13 +423,46 @@ def logical_xor(
     )
 
 
+def bitwise_and(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, bool, Sequence[Union[int, bool]]],
+    rhs_val: Union[TRTTensor, int, bool, Sequence[Union[int, bool]]],
+) -> TRTTensor:
+    return logical_and(ctx, target, source_ir, f"{name}_logical_and", lhs_val, rhs_val)
+
+
+def bitwise_or(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, bool, Sequence[Union[int, bool]]],
+    rhs_val: Union[TRTTensor, int, bool, Sequence[Union[int, bool]]],
+) -> TRTTensor:
+    return logical_or(ctx, target, source_ir, f"{name}_logical_or", lhs_val, rhs_val)
+
+
+def bitwise_xor(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, bool, Sequence[Union[int, bool]]],
+    rhs_val: Union[TRTTensor, int, bool, Sequence[Union[int, bool]]],
+) -> TRTTensor:
+    return logical_xor(ctx, target, source_ir, f"{name}_logical_xor", lhs_val, rhs_val)
+
+
 def eq(
     ctx: ConversionContext,
     target: Target,
     source_ir: Optional[SourceIR],
     name: str,
-    lhs_val: Union[TRTTensor, int, float],
-    rhs_val: Union[TRTTensor, int, float],
+    lhs_val: TRTTensor,
+    rhs_val: Union[TRTTensor, int, float, bool, Sequence[Union[int, float, bool]]],
```

Comment:
In what cases could this be a sequence of integers, floats, or bools? Could the `convert_binary_elementwise` function handle such situations?

Reply:
Thanks for the comments. At first I thought it could be a sequence of integers, floats, or bools, but I checked the doc it should not be.